dockstore-client/pom.xml

```diff
@@ -175,7 +175,11 @@
             <artifactId>logback-classic</artifactId>
             <optional>true</optional>
         </dependency>
-
+        <dependency>
```

Comment:
Take out the dependency on a slack API. 
Keep in mind that the request wasn't to make something for Slack integration, rather that Slack provides an example endpoint that can take HTTP POSTS for testing. You can use something as simple as CURL https://api.slack.com/incoming-webhooks (in Java, HttpClient)